packages/flutter/test/cupertino/segmented_control_test.dart

```diff
@@ -893,7 +919,13 @@ void main() {
       ),
     );
 
-    await expectLater(find.byType(RepaintBoundary), matchesGoldenFile('segmented_control_test.0.0.png'));
+    // if golden files are regenerated on a non-Mac machine,
+    // change the 'skip' line to only run this test on that type of machine
```

Comment:
Explain why we are only running this on Mac?

Reply:
I have added the explanation to the comment.